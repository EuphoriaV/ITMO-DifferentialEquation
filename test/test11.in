1.98
((7.92 - x) * ((x - 3.23) * 2.6))
3.57
3.51