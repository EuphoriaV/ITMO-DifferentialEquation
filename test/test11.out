0.0063359